fix: lg file mess up

zhixzhan opened the pull request:
## Description

<!---
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

If this is a bug fix, please describe the root cause and analysis of this problem.
---->

## Task Item

close #3136 

<!---
Please include a link to the related issue. [Ex. `Closes #<issue #>`](https://help.github.com/en/articles/closing-issues-using-keywords)
---->

## Screenshots

<!---
Please include screenshots or gifs if your PR include UX changes.
--->

boydc2014 commented on Composer/packages/server/src/models/bot/botProject.ts:
```diff
@@ -778,16 +778,6 @@ export class BotProject {
 
     for (const file of files) {
       const { realFilePath, content } = file;
-      await this.fileStorage.removeFile(realFilePath);
```
Remove this dangerous delete and write back is ok, but i think we can take the chance to totally disable this whole "tranformer" work for now,  which is done somewhat very ad-hoc. 

One top thing in R10 is about version, we should take the transformer work with that to have a better transformer. 

What do you think @cwhitten

zhixzhan replied:
I agree remove  those two "transformer" methods, they already exist about 2 month on master, user's bot should already be migrated.

zhixzhan commented:
> Please also update the lg package to fix the parsing issue.

Sure, there are some upgrade issue haven't resolve, added `WIP` tag to hold this PR.

boydc2014 commented:
> > Please also update the lg package to fix the parsing issue.
> 
> Sure, there are some upgrade issue haven't resolve, added `WIP` tag to hold this PR.

FYI, the js fix is in master, you should be unblocked now

zhixzhan commented:
> > > Please also update the lg package to fix the parsing issue.
> > 
> > 
> > Sure, there are some upgrade issue haven't resolve, added `WIP` tag to hold this PR.
> 
> FYI, the js fix is in master, you should be unblocked now

updated.
